self_hosting_machinery/webgui/selfhost_fastapi_completions.py

```diff
@@ -285,6 +322,12 @@ async def _coding_assistant_caps(self):
             "running_models": models_available,
             "code_completion_default_model": code_completion_default_model,
             "code_chat_default_model": code_chat_default_model,
+
+            "default_embeddings_model": embeddings_default_model,
```

Comment:
default_embeddings_model -> embeddings_default_model

Reply:
Oleg asked spefically for default_embeddings_model. Previously it was as you proposed

Reply:
doesn't look like problem to change it in refact-lsp